pkg/runtime/runtime.go

```diff
@@ -387,6 +386,8 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 
 	a.loadAppConfiguration()
 
+	go a.processComponents()
```

Comment:
What is it for?

Reply:
after get app configuration, then start process components. so runtime can judge which module can be process.